sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/TestUtils.java

```diff
@@ -172,7 +171,7 @@ static CustomFormModel getExpectedUnlabeledModel() {
             }
         };
         CustomFormSubModel customFormSubModel = new CustomFormSubModel(null, fieldMap, "form-0");
-        return new CustomFormModel(VALID_MODEL_ID, CustomFormModelStatus.READY,
+        return new CustomFormModel("95537f1b-aac4-4da8-8292-f1b93ac4c8f8", CustomFormModelStatus.READY,
```

Comment:
create/use a\ static final variable for it.

Reply:
It isn't reused anywhere else so didn't bother putting it in a static variable.